src/main/webapp/js/instructor.js

```diff
@@ -14,6 +14,11 @@ $(document).ready(function() {
     
     // bind the show picture onhover events
     bindStudentPhotoHoverLink('.profile-pic-icon-hover');
+    
+    /* Formatting searchKey for highlight searchKey plugin ( ['string1','string2',...] )*/
+    var searchKey = $('#searchBox').val();
+    var splittedSearchKey = searchKey.split(" ");
+    $(".panel-body").highlight(splittedSearchKey);
```

Comment:
Be consistent with your string quotations, use ```' '``` instead of ```" "``` for example - note that that is also our style convention